examples/tutorials/api/irfs.py

```diff
@@ -0,0 +1,458 @@
+    data[i][-1] = data[i][0] * 1.2
+
+
+aeff_3d = EffectiveArea3D(
+    [energy_axis, fov_lon_axis, fov_lat_axis], data=data, unit=u.m**2
+)
+print(aeff_3d)
+
+res = aeff_3d.evaluate(
+    fov_lon=[-0.5, 0.8] * u.deg,
+    fov_lat=[-0.5, 1.0] * u.deg,
+    energy_true=[0.2, 8.0] * u.TeV,
+)
+print(res)
+
+# to visualise at a given energy
+# sphinx_gallery_thumbnail_number = 1
+aeff_eval = aeff_3d.evaluate(energy_true=[1.0] * u.TeV)
+
+ax = plt.subplot()
+with quantity_support():
+    caxes = ax.pcolormesh(
+        fov_lat_axis.edges, fov_lon_axis.edges, aeff_eval.value.squeeze()
+    )
+fov_lat_axis.format_plot_xaxis(ax)
+fov_lon_axis.format_plot_yaxis(ax)
+ax.set_title("Asymmetric effective area")
+
+
+######################################################################
+# Serialisation
+# ~~~~~~~~~~~~~
+#
+# For serialisation, we need to add the class definition to the
+# `IRF_DL3_HDU_SPECIFICATION` dictionary
+#
+
+IRF_DL3_HDU_SPECIFICATION["aeff_3d"] = {
+    "extname": "EFFECTIVE AREA",
+    "column_name": "EFFAREA",
+    "mandatory_keywords": {
+        **COMMON_IRF_HEADERS,
+        "HDUCLAS2": "EFF_AREA",
+        "HDUCLAS3": "FULL-ENCLOSURE",  # added here to have HDUCLASN in order
+        "HDUCLAS4": "AEFF_3D",
+    },
+}
+
+aeff_3d.write("test_aeff3d.fits", overwrite=True)
+
+aeff_new = EffectiveArea3D.read("test_aeff3d.fits")
+print(aeff_new)
+
+
+######################################################################
+# Create exposure map
+# ~~~~~~~~~~~~~~~~~~~
+#
+# DL4 data products can be created from these IRFs.
+#
+
+axis = MapAxis.from_energy_bounds(0.1 * u.TeV, 10 * u.TeV, 6, name="energy_true")
+pointing = SkyCoord(2, 1, unit="deg")
+geom = WcsGeom.create(npix=(4, 3), binsz=2, axes=[axis], skydir=pointing)
+
+print(geom)
+
+exposure_map = make_map_exposure_true_energy(
+    pointing=pointing, livetime="42 h", aeff=aeff_3d, geom=geom
+)
+
+exposure_map.plot_grid(add_cbar=True)
+
+
+######################################################################
+# Energy Dispersion
+# -----------------
+#
+
+
+class EnergyDispersion3D(IRF):
+    tag = "edisp_3d"
+    required_axes = ["energy_true", "migra", "fov_lon", "fov_lat"]
+    default_unit = u.one
+
+    @classmethod
+    def from_gauss(
+        cls,
+        energy_axis_true,
+        migra_axis,
+        fov_lon_axis,
+        fov_lat_axis,
+        bias,
+        sigma,
+        pdf_threshold=1e-6,
+    ):
+        axes = MapAxes([energy_axis_true, migra_axis, fov_lon_axis, fov_lat_axis])
+        coords = axes.get_coord(mode="edges", axis_name="migra")
+
+        migra_min = coords["migra"][:, :-1, :]
+        migra_max = coords["migra"][:, 1:, :]
+
+        # Analytical formula for integral of Gaussian
+        s = np.sqrt(2) * sigma
+        t1 = (migra_max - 1 - bias) / s
+        t2 = (migra_min - 1 - bias) / s
+        pdf = (scipy.special.erf(t1) - scipy.special.erf(t2)) / 2
+        pdf = pdf / (migra_max - migra_min)
+
+        r1 = np.rollaxis(pdf, -1, 1)
```

Comment:
As this function is exposed, can you add a simple comment to explain what do these 2 lines?

Reply:
Removed now